src/preamble.js

```diff
@@ -2032,4 +2032,343 @@ Module['FS_createPreloadedFile'] = FS.createPreloadedFile;
 var cyberDWARFFile = '{{{ BUNDLED_CD_DEBUG_FILE }}}';
 #endif
 
+#if BINARYEN
+function integrateWasmJS(Module) {
```

Comment:
Almost none, except for the necessary changes in formatting (using emscripten's preprocessor instead of the hackish stuff we had before, so now we just use `BINARYEN_METHOD` directly for example instead of python code doing a special search&replace).